docs/xml/nginx/changes.xml

```diff
@@ -5,6 +5,73 @@
 <change_log title="nginx">
 
 
+<changes ver="1.27.2" date="2024-10-02">
+
+<change type="feature">
+<para lang="ru">
+SSL-сертификаты, секретные ключи и списки CRL теперь кешируются
+при загрузке на старте или во время переконфигурации.
+</para>
+<para lang="en">
+loading SSL certificates, secret keys, and CRLs is now cached
+on start or during reconfiguration.
+</para>
+</change>
+
+<change type="feature">
+<para lang="ru">
+проверка клиентских сертификатов с помощью OCSP в модуле stream.
+</para>
+<para lang="en">
+client certificate validation with OCSP in the stream module.
+</para>
+</change>
+
+<change type="feature">
+<para lang="ru">
+поддержка OCSP stapling в модуле stream.
+</para>
+<para lang="en">
+OCSP stapling support in the stream module.
+</para>
+</change>
+
+<change type="feature">
+<para lang="ru">
+директива proxy_pass_trailers в модуле ngx_http_proxy_module.
+</para>
+<para lang="en">
+the "proxy_pass_trailers" directive in the ngx_http_proxy_module.
+</para>
+</change>
+
+<change type="feature">
+<para lang="ru">
+директива ssl_client_certificate теперь поддерживает сертификаты
+с дополнительными данными в OpenSSL-специфичном PEM-формате
+"TRUSTED CERTIFICATE".
+</para>
+<para lang="en">
+the "ssl_client_certificate" directive now supports certificates
+with auxiliary information in the OpenSSL specific PEM format
+"TRUSTED CERTIFICATE".
```

Comment:
I couldn't find anyone using `PEM format "TRUSTED CERTIFICATE"`. Maybe the whole OpenSSL appendix makes no sense and the text should end with just  "...the auxiliary information". WDYT?

Reply:
IMHO, this wording sounds confusing, but up to you.